sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/DocumentCacheTest.java

```diff
@@ -13,29 +13,36 @@
 import com.microsoft.appcenter.utils.storage.SQLiteUtils;
 
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.mockito.AdditionalMatchers;
 import org.mockito.ArgumentCaptor;
 import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.rule.PowerMockRule;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 import static org.powermock.api.mockito.PowerMockito.mockStatic;
 import static org.powermock.api.mockito.PowerMockito.whenNew;
 
 @SuppressWarnings("unused")
-@RunWith(PowerMockRunner.class)
 @PrepareForTest({SQLiteUtils.class, AppCenterLog.class, DatabaseManager.class, DocumentCache.class})
 public class DocumentCacheTest {
 
+    static final String PARTITION = "partition";
```

Comment:
If you were using android studio commit dialog or looking at warnings, it would tell you to use private here.

Reply:
<img width="1044" alt="screen shot 2019-03-07 at 2 44 42 pm" src="https://user-images.githubusercontent.com/623672/53994587-94d32780-40e7-11e9-8725-3fa277d4e2d0.png">

Sorry to insist but please use the tools.